tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigSpec.scala

```diff
@@ -87,6 +91,77 @@ class SearchConfigSpec extends BaseSpec {
       }
     }
 
+    "have the correct name property from schema:name" in {
+      val query = queryField(neuronMorphologyId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "sm080522a1-5_idA" }"""
+      }
+    }
+
+    "have the correct name property from rdfs:label" in {
+      val query = queryField(neuronDensityId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "Neuron density: CA1" }"""
+      }
+    }
+
+    "have the correct name property from skos:prefLabel" in {
+      val query = queryField(traceId, "name")
+      assertOneSource(query) { json =>
+        json shouldEqual json"""{ "name": "S1J_L6_IPC_cADpyr_2" }"""
+      }
+    }
```

Comment:
I think this is a general problem with these tests, but I'll write my example here

I would look at this and wonder what it's asserting. Perhaps another way that this could be written
```
"have a name field when the resource only has skos:prefLabel" in {
  assertOneSource(queryField(traceId, "name")) {
    json should have field("name", traceSkosPrefLabel)
  }
}
```
I think the test name is the only place where you can explain why the name property should have a certain value.
"S1J_L6_IPC_cADpyr_2" is a magic value, but by labelling it you can explain what it really is

Reply:
I won't make changes now but we should soon enough have updates to do there so and we can rework tests then